### ISSUE TITLE
1.3.1 breaking undefined reference

### ISSUE THREAD

**Issue by kjgorman** [shoreditch-ops/minigun#37]
Hi @hassy the 1.3.1 version you pushed earlier has a rather nasty breaking bug in it.

```
➜  repro  minigun run repro.json
/usr/local/lib/node_modules/minigun/node_modules/minigun-core/lib/runner.js:67
          fields: script.config.payload.fields,
                                       ^

TypeError: Cannot read property 'fields' of undefined
    at runner (/usr/local/lib/node_modules/minigun/node_modules/minigun-core/lib/runner.js:67:40)
    at /usr/local/lib/node_modules/minigun/lib/commands/run.js:149:16
    at /usr/local/lib/node_modules/minigun/node_modules/async/lib/async.js:52:16
    at Immediate._onImmediate (/usr/local/lib/node_modules/minigun/node_modules/async/lib/async.js:1206:34)
    at processImmediate [as _immediateCallback] (timers.js:383:17)
```

Here is a simple reproduction config:

```
{
  "config": {
      "target": "https://google.com",
      "phases": [
        { "duration": 120, "arrivalRate": 10 }
      ]
  },
  "scenarios": [
    { "get": { "url": "/ping" } }
  ]
}
```

The bug appears to be because of this line: https://github.com/shoreditch-ops/minigun/blob/46ddfd1c9f3efa57a171287719c44216020ca3be/lib/commands/run.js#L124 causing this falsity check: https://github.com/shoreditch-ops/minigun-core/blob/380f5c76f86ea6ca1d14a861bf819e3651373401/lib/runner.js#L63 to incorrectly enter the block. Then on L67 referencing `payload.fields` fails because `payload` does not actually exist.

**Comment by kjgorman** [comment]
(a cursory `context.payload = []` appears to fix this simple case. I don't have something that uses payloads handy to test that though.)

**Comment by hassy** [comment]
@kjgorman oops! thanks for the report. Just published `1.3.2` with a fix.
